src/shared/App.less

```diff
@@ -9,27 +9,40 @@
   margin: 52px auto 0;
   display: flex;
   background: #f5f5f5 !important;
+
   &.-unconstrained-width {
     padding: 2em;
     max-width: none;
   }
+
   &.resource-view {
-    max-width: 60%;
+    max-width: 1320px;
```

Comment:
this is just a max-width so if the screen is too small then the page will be okey, this meant for big screen then it will take only `1320px`

Reply:
the 60% was used for the `GalleryView` when the user click on the resource and open the drawer, if the user access the resource (after refresh or by going directly to the resource) then it will be the same `60%` of the screen, I see this behavior many times and it's annoying for the user, this is why I make the resource view take `100%` of the screen and at max `1320px`